src/main/java/org/opentripplanner/RideHailingServicesParameters.java

```diff
@@ -0,0 +1,8 @@
+package org.opentripplanner;
+
+import java.util.List;
+import org.opentripplanner.ext.ridehailing.RideHailingServiceParameters;
```

Comment:
Dependency to `ext`- only allowed in configuration packages.

Reply:
This whole file is gone in 16e312aca23dd8500fba42e5ebb3b171fadb75fc